code/datums/brain_damage/imaginary_friend.dm

```diff
@@ -138,17 +154,68 @@
 	to_chat(owner, "[rendered]")
 	to_chat(src, "[rendered]")
 
+	//speech bubble
+	if(owner.client)
+		var/image/I = image('icons/mob/talk.dmi', src, "default[say_test(message)]", FLY_LAYER)
```

Comment:
why not a mutable_appearance?

Reply:
would it work? i probably copied code that predated it

Reply:
Bump, everything else looks good.

Reply:
Damn i always forget when there's a pending request, will fix asap